pvmismatch/contrib/ModuleMismatchSimulator.py

```diff
@@ -0,0 +1,510 @@
+'''
```

Comment:
It's showing up as executable, `-rwxr-xr-x` (= 755) on my machine, but I'm on windows, so not sure if it is on yours too.
![already_executable](https://cloud.githubusercontent.com/assets/1385621/22844452/37e75be2-ef93-11e6-940d-c28df6e33073.png)